apps/examples/testcase/ta_tc/messaging/utc/utc_messaging_multicast.c

```diff
@@ -37,6 +37,7 @@
 
 static bool tc_multi_chk = TC_OK;
 static int tc_multi_count = 0;
+static bool tc_cleanup_chk = false;
```

Comment:
UTC being used to check whole of test is completed successfully or failed.
Then I think you can use only two type, int g_multi_count & int g_result (OK or ERROR instead of ret)
Also it is static, you'd better initialize when tc started.